tests/test_status_change.py

```diff
@@ -488,9 +521,23 @@ async def test_ci_passed_and_check_run_timed_out_awaiting_merge_label_pr_is_not_
         },
     }
 
-    gh = FakeGH(getitem=getitem)
+    getiter = {
+        "/repos/python/cpython/pulls/5547/commits": [
+            {
+                "sha": "f2393593c99dd2d3ab8bfab6fcc5ddee540518a9",
+                "commit": {
+                    "message": "bpo-32720: Fixed the replacement field grammar documentation. (GH-5544)\n\n`arg_name` and `element_index` are defined as `digit`+ instead of `integer`.\n(cherry picked from commit 7a561afd2c79f63a6008843b83733911d07f0119)\n\nCo-authored-by: Mariatta <Mariatta@users.noreply.github.com>"
+                },
+            }
+        ]
+    }
```

Comment:
This object seem duplicated several time.  Can it be defined once and reused?  If not, could the `"message"` be trimmed and the object made more compact so that it takes 3 lines instead of several?

Reply:
The pattern in these tests seems to be redefining. Objects like `items` could also be re-used.

Do you want me to refactor? Or follow the current style with a shorter message?

Reply:
Perhaps we could refactor in a separate PR.